src/java/fr/paris/lutece/util/annotation/ScannotationDB.java

```diff
@@ -127,7 +127,7 @@ public void init( )
             {
                 if ( AppLogService.isDebugEnabled( ) )
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.